Week-2/Exercise2.1/css/style.css

```diff
@@ -0,0 +1,250 @@
+:root {
+  --fontcolor: #4d3232;
+  --greybackground: #f1f1f1;
+  --footercolor: #6495ed;
+  --whitefont: #ffffff;
+  --whitebackground: #ffffff;
+  --linkcolor: #1358d0;
+  --linkhovercolor: #ff0000;
+  --headercolor: #003c68;
+  --formcolor: #b1c4c5;
+  --buttoncolor: #374151;
+  --buttonhover: #009999;
```

Comment:
```
  --whitefont: #ffffff;
  --whitebackground: #ffffff;
```

Instead of two variables having the same property, create one common variable such as `--white-color: #fff` and reuse it wherever necessary.